app/swagger/requests/prescriptions.rb

```diff
@@ -0,0 +1,306 @@
+                    description: 'number of results, between 1 and 99'
+
+          parameter name: :sort, in: :query, required: false, type: :string,
+                    description: "Comma separated sort field(s), prepend field(s) with '-' for descending sort"
+
+          parameter name: :filter, in: :query, required: false, type: :string,
+                    description: 'Filter on refill_status: [[refill_status][logical operator]=status]'
+
+          response 200 do
+            key :description, 'prescription index response'
+
+            schema do
+              key :'$ref', :PrescriptionsIndexResponse
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/prescriptions/active' do
+        operation :get do
+          key :description, 'Get a list of active prescriptions'
+          key :operationId, 'indexActivePrescriptions'
+          key :tags, %w(prescriptions)
+
+          parameter name: :page, in: :query, required: false, type: :integer,
+                    description: 'Page of results, greater than 0'
+
+          parameter name: :per_page, in: :query, required: false, type: :integer,
+                    description: 'number of results, between 1 and 99'
+
+          parameter name: :sort, in: :query, required: false, type: :string,
+                    description: "Comma separated sort field(s), prepend field(s) with '-' for descending sort"
+
+          response 200 do
+            key :description, 'active prescription index response'
+
+            schema do
+              key :'$ref', :PrescriptionsIndexResponse
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/prescriptions/{id}' do
+        operation :get do
+          key :description, 'Get details about a prescription'
+          key :operationId, 'showPrescriptions'
+          key :tags, %w(prescriptions)
+
+          parameter name: :id, in: :path, required: true, type: :integer, description: 'id of the presecription'
+
+          response 200 do
+            key :description, 'prescription show response'
+
+            schema do
+              key :'$ref', :PrescriptionsPrescription
+            end
+          end
+
+          response 404 do
+            key :description, 'Record not available'
+
+            schema do
+              key :'$ref', :PrescriptionError
+            end
+          end
+        end
+      end
+
+      swagger_path '/v0/prescriptions/{id}/refill' do
+        operation :patch do
+          key :description, 'refills a prescription'
+          key :operationId, 'refillPrescriptions'
+          key :tags, %w(prescriptions)
+
+          parameter name: :id, in: :path, required: true, type: :integer, description: 'id of the presecription'
+
+          response 204 do
+            key :description, 'prescription refill response'
+          end
+
+          response 400 do
+            key :description, 'prescription not available'
+
```

Comment:
I actually think there is another type of error that could occur or "prescription not available" the only one?